Magitek/Logic/BlackMage/SingleTarget.cs

```diff
@@ -315,10 +323,15 @@ public static async Task<bool> Blizzard3()
             //Post 72 logic
             if (Core.Me.ClassLevel > 71)
             {
+                if (await Spells.ManaFont.Cast(Core.Me))
```

Comment:
Seems like this is duplicated, inside the ClassLevel check and after it. Could place it once above the if statement?